setup.py

```diff
@@ -137,9 +137,15 @@ def get_release_info():
     'Topic :: Software Development :: Widget Sets',
 ]
 
+# Unfortunately, distutils.version.StrictVersion does not support semantic
+# versioning (www.semver.org) for pre-releases. This breaks, e.g,  bdist_msi
+# As a workaround, we normalize the version to be acceptable for StrictVersion
+# Note that the normalized version is identical to the semver except for
+# pre-releases (where '-dev' is replaced by 'a0').
+_version = release.version.replace('-dev', 'a0')
```

Comment:
After performing some tests we have observed that `X.Y.Z-alpha` is normalized by distutils to a MSI compatible version number `X.Y.Za0`. Using this is still compatible with *semver* and does not require any special replacements for the `distutils`. Another option would be to use `X.Y.Z-a0` that would also normalize to `X.Y.Za0`.